src/transformers/models/cohere/modeling_cohere.py

```diff
@@ -61,20 +57,7 @@
 _CONFIG_FOR_DOC = "CohereConfig"
 
 
-# Copied from transformers.models.llama.modeling_llama._get_unpad_data
```

Comment:
that's a problem no? the unpad_data should still be present!

Reply:
For some reason it has been pasred below .. https://github.com/huggingface/transformers/pull/31211/files#r1647528352